tests/unit_tests/test_dqc_us_0005.py

```diff
@@ -128,6 +128,68 @@ def test_axis_exists_multi_axis(self):
             facts.axis_exists(mock_val, mock_fact, 'not a used axis')
         )
 
+    def test_run_checks_EntityCommonStockSharesOutstanding(self):
+        msg = messages.get_message('DQC.US.0005', "17")
```

Comment:
SG: Single line testing docstring missing. ([REF](https://github.com/Workiva/Stylin/blob/master/STYLEGUIDE.rst#docstrings-and-unit-tests))